packages/opentelemetry-resources/src/types.ts

```diff
@@ -15,15 +15,13 @@
  */
 
 import { ResourceDetectionConfig } from './config';
-import { SpanAttributes } from '@opentelemetry/api';
+import { Attributes } from '@opentelemetry/api';
 import { IResource } from './IResource';
 
 /**
  * Interface for Resource attributes.
- * General `Attributes` interface is added in api v1.1.0.
- * To backward support older api (1.0.x), the deprecated `SpanAttributes` is used here.
  */
-export type ResourceAttributes = SpanAttributes;
+export type ResourceAttributes = Attributes;
```

Comment:
Would you mind adding a TODO comment to remove the usage of `ResourceAttributes` as well?

Reply:
done https://github.com/open-telemetry/opentelemetry-js/pull/4428/commits/3792a7b0ebfe4d2f0bc49e6728a1772e466098e9